woocommerce/woocommerce-android
# Add toolbar for BaseAddressEditingFragment

Part of #10570 

### Description

This PR introduces individual toolbars for BaseAddressEditingFragment screen, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the Billing and Shipping address screens (Order list -> Order detail -> Shipping/Billing address) and ensure the toolbar and its action work as expected on both phone and tablet.

### Images/gif

<img width="433" alt="Screenshot 2024-02-12 at 5 43 41 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/94f2a6be-2f83-4e7d-905d-486f97785914">
